### ISSUE TITLE
Separate build target or enviroment for publishing the style guide

### ISSUE THREAD

**Issue by rajasegar** [chrislopresto/ember-freestyle#156]
I love the generated living style guide at the `/freestyle` route , but what if i just want the styleguide not the entire app with all the api calls, initializers, ember data related operations, i tried using a an entirely new enviroment config, but it doesn't work. Any directions/ pointers would be very much appreciated.

**Comment by sivakumar-kailasam** [comment]
@rajasegar checkout https://simplabs.com/blog/2018/01/24/ember-freestyle.html. If you can please try raising a PR linking to the article on this repo's docs :)

**Comment by rajasegar** [comment]
@sivakumar-kailasam Yes, i have already seen that article, its pretty much a starting point of what i am trying to accomplish, but what it is essentially missing is how to stop the api calls from happening, muting the initializers and ember-data , that's what i am more concerned with.

Since one of the first constraints for me is keeping the styleguide code as part of the original product codebase, otherwise the whole purpose is lost for the living styleguide.

**Comment by bertdeblock** [comment]
👋 Going through some old issues.

We've also had a similar challenge at our company and the way we've solved it is to have a separate addon that includes all UI components and the style guide itself. The dummy app of the addon renders the style guide and can easily be deployed separately.

Hope this helps!

Going to close for now if that's okay as this is more of question and not something `ember-freestyle` can/should solve I feel.
